src/go/rpk/pkg/utils/files.go

```diff
@@ -21,6 +21,28 @@ import (
 	"github.com/spf13/afero"
 )
 
+func ReadFileLinesBinary(fs afero.Fs, filePath string) ([]byte, error) {
```

Comment:
I don't think this is necesssary. This does the same as the function below, `ReadFileLines`, but returns a `[]byte` instead of a `[]string`. You could just use the existing ones and cast the bytes to strings. Also, the `[]byte` seems a little odd. I would expect a `[][]byte`, since you're splitting it by lines.

Reply:
change it, I use `afero.ReadFile` now